software/url-to-png.yml

```
@@ -0,0 +1,12 @@
+name: "URL-to-PNG"
+website_url: "https://github.com/jasonraimondi/url-to-png"
+source_code_url: "https://github.com/jasonraimondi/url-to-png"
+description: "URL to PNG utility featuring parallel rendering using Playwright for screenshots and with storage caching via Local, S3, or CouchDB."
+licenses:
+  - MIT
+platforms:
+  - Nodejs
+  - Docker
```

```suggestion
platforms:
  - Docker
```

Docker is the only supported installation method

---

I've updated the documentation to include hosting via nodejs.

Please see
- https://github.com/jasonraimondi/url-to-png?tab=readme-ov-file#local-serve
- https://jasonraimondi.github.io/url-to-png/getting_started/#local-development

---

Thanks